draft-regext-rdap-extensions.md

```diff
@@ -396,24 +429,28 @@ JSON names:
       }
     }
 
-It is RECOMMENDED that object class names comprise lowercase ASCII
-characters, and that the "\_" (underscore) character be used as a word
-separator.  Though "objectClassName" is a string and [@!RFC9083] does
+It is RECOMMENDED that object class names use the "camel case" style
+described in (#camel_casing).
+Though "objectClassName" is a string and [@!RFC9083] does
 define one object class name with a space separator (i.e. "ip
 network"), usage of the space character or any other character that
 requires URL-encoding is NOT RECOMMENDED.  When object class names are
```

Comment:
I think removing the sentence that begins with "When object class names ..." would be good, per Pawel's comment that this is mandated by the HTTP specifications anyway, and also because the text makes it sound like encoding is required in non-URL contexts.